airbyte-integrations/connectors/source-mongodb-new/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'application'
+    id 'airbyte-docker'
+    id 'airbyte-integration-test-java'
+}
+
+application {
+    mainClass = 'io.airbyte.integrations.source.mongodb.MongoDbSource'
+}
+
+dependencies {
+    implementation project(':airbyte-db:lib')
+    implementation project(':airbyte-integrations:bases:base-java')
+    implementation project(':airbyte-protocol:models')
+    implementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
+    implementation project(':airbyte-integrations:connectors:source-relational-db')
+    implementation group: 'org.mongodb', name: 'mongo-java-driver', version: '3.12.8'
```

Comment:
Do we really need the driver here? Is there something outside `airbyte-db` and `MongodbDatabase`?

Reply:
Yes, there is MongoCollection, BsonType and Document classes from mongo